### PR TITLE
introduce import syntax (only the syntax)

### PR DESCRIPTION
**Pull request by rayshih**
check the issue #41 for in-depth discussion

### CODE REVIEW

File: src/Violet/Core.idr [hunk 1]
```diff
@@ -35,6 +35,7 @@ eval env tm = case tm of
   Pi x a b => VPi x (eval env a) (\u => eval (extend env x u) b)
   Let x a t u => eval (extend env x (eval env t)) u
   Postulate x a u => eval (extend env x (VVar x)) u
+  Import name u => eval env u
```
Comment (rayshih) [inline src/Violet/Core.idr:38]: I'm actually not sure if we should make this into core language.

If we do this, then import syntax should be easily supported in repl, but surely complicated the core

If we don't do this, then we need to figure out a way to resolve this before inference. And probably the imports need to be before all top level decls in a module.

Comment (dannypsnl) [reply src/Violet/Core.idr:38]: I think it **should not** be core, we can have one more layer call **command**, than have the concept like `:t term` in REPL will be natural now.

Thus, we can have **import Xxx** as a command.

Comment (dannypsnl) [reply src/Violet/Core.idr:38]: Ok, I'm more wide awake now. The point is import is not a core term, it will belong to ctx & env.

Comment (dannypsnl) [reply src/Violet/Core.idr:38]: That's say we have a `(ctx, env)` for each module, the REPL is a `Has [State REPL (ctx, env)] e`, and import statement will encapsulate another module's `(ctx, env)` as a special structure (by hiding non-exported definitions)

**Pull request closed without merging**